### PR TITLE
feat: Brazilian Portuguese internationalization

### PR DESCRIPTION
**Pull request by danielmichalichyn**
### Definition of Ready
This merge add's i18n for Brazilian Portuguese. I've based myself on others i18n merge requests. If you find any issues related to the code, please let me know! 
Thanks in advance.

- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: console/src/assets/i18n/bg.json [hunk 1]
```diff
@@ -1034,7 +1034,8 @@
       "ja": "日本語",
       "pl": "Полски",
       "zh": "简体中文",
-      "bg": "Български"
+      "bg": "Български",
+      "pt": "португалски"
```
Comment (hifabienne) [inline console/src/assets/i18n/bg.json:1038]:
```suggestion
      "pt": "Portuguese"
```

File: console/src/assets/i18n/bg.json [hunk 2]
```diff
@@ -1234,7 +1235,8 @@
         "ja": "日本語",
         "pl": "Полски",
         "zh": "简体中文",
-        "bg": "Български"
+        "bg": "Български",
+        "pt": "португалски"
```
Comment (hifabienne) [inline console/src/assets/i18n/bg.json:1239]:
```suggestion
        "pt": "Portuguese"
```

File: console/src/assets/i18n/bg.json [hunk 3]
```diff
@@ -2101,7 +2103,8 @@
     "ja": "日本語",
     "pl": "Полски",
     "zh": "简体中文",
-    "bg": "Български"
+    "bg": "Български",
+    "pt": "португалски"
```
Comment (hifabienne) [inline console/src/assets/i18n/bg.json:2107]:
```suggestion
    "pt": "Portuguese"
```

Comment (hifabienne) [inline console/src/assets/i18n/bg.json:2107]:
```suggestion
    "pt": "Portuguese"
```

File: internal/api/ui/login/static/i18n/pt.yaml [hunk 4]
```diff
@@ -0,0 +1,446 @@
...
+  PrivacyLinkText: política de privacidade
+  German: Alemão
+  English: Inglês
+  Italian: Italiano
+  French: Francês
+  Chinese: Chinês simplificado
+  Polish: Polonês
+  Japanese: Japonês
+  Spanish: Espanhol
+  Bulgarian: Búlgaro
```
Comment (hifabienne) [inline internal/api/ui/login/static/i18n/pt.yaml:316]: Please add portugese here

File: console/src/assets/i18n/zh.json [hunk 5]
```diff
@@ -1040,7 +1040,8 @@
       "ja": "日本語",
       "pl": "Polski",
       "zh": "简体中文",
-      "bg": "Български"
+      "bg": "Български",
+			"pt": "Portuguese"
```
Comment (grobarko) [inline console/src/assets/i18n/zh.json:1044]: formatting

File: internal/api/ui/login/static/i18n/pt.yaml (hunk 4 above)
Comment (danielmichalichyn) [reply internal/api/ui/login/static/i18n/pt.yaml:317]: Hi @hifabienne , i've added portuguese. Thanks for the input!

**Comment by hifabienne** [comment]
@danielmichalichyn Can you please merge the main branch back to your branch? We did have another contributions in the languages, that now conflicts a little with yours. Thanks in advanced

**Comment by danielmichalichyn** [comment]
> @danielmichalichyn Can you please merge the main branch back to your branch? We did have another contributions in the languages, that now conflicts a little with yours. Thanks in advanced

Done @hifabienne, i've take the time to add the i18n Portuguese properties to the Macedonian files as well :) If you find any more issues, i'm glad to help.

**Review by hifabienne — APPROVED** [review]

**Comment by hifabienne** [comment]
hei @danielmichalichyn 
Thank you for your contribution we really appreciate it.
To thank you we would like to send you some ZITADEL swag. If you are interested please send an email to hi@zitadel.com

**Pull request merged**